components/Divider.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export default function Divider() {
+    return <hr name="divider" role="none" />;
+  }
+
+  const Hr = styled.hr`
+  border: none;
+  width: 100%;
+  margin: 0;
+  height: 1px;
+  background: color-water-10;
+`;
```

Comment:
you make a styled hr here but you are using a normal one
i guess you need to change the hr in the return to Hr

Reply:
Hi Nina, thanks for the feedback! Good point! I had totally overlooked it. Thanks